Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/example/QueueingTask.kt

```diff
@@ -74,4 +77,11 @@ open class QueueingTask @Inject constructor() : Task<DefaultProgress, QueueingTa
         override suspend fun createConfig(): TaskFactory.Config = Config()
         override val taskProvider: () -> Task<DefaultProgress, Result> = { taskByDagger.get() }
     }
+
+    companion object {
+        suspend fun isPreconditionMet(): Boolean{
+            return preconditionIsMet
+        }
+        var preconditionIsMet = true
```

Comment:
Moreover, this is only a `var` instead of a `val` becaus of the Test afaik? How about annotating it accordingly.